primitives/parachain-inherent/src/mock.rs

```diff
@@ -178,6 +178,14 @@ impl InherentDataProvider for MockValidationDataInherentDataProvider {
 			sproof_builder.upsert_inbound_channel(*para_id).mqc_head = Some(channel_mqc.head());
 		}
 
+		// Epoch is set equal to current para block
+		sproof_builder.current_epoch = self.current_para_block.into();
```

Comment:
Should we not make this configurable to the outside? I find it weird to use the current para block for this.

Reply:
I made the epoch configurable in terms of the number of parachain blocks per relay chain epoch. Let me know if you had something else in mind.